src/a11y-dialog.ts

```diff
@@ -307,30 +307,49 @@ function isFocusable(el: HTMLElement) {
  */
 function trapTabKey(el: HTMLElement, event: KeyboardEvent) {
   const focusableChildren = getFocusableChildren(el)
-  const focusedItemIndex = focusableChildren.indexOf(
-    document.activeElement as HTMLElement
-  )
+  const firstFocusableChild = focusableChildren[0]
+  const lastFocusableChild = focusableChildren[focusableChildren.length - 1]
```

Comment:
Side note, now that we no longer supper Internet Explorer, we could use `.at(-1)` here to shorten this line.

https://caniuse.com/mdn-javascript_builtins_array_at